radio/src/main/res/values/style.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <style name="Widget.Vitamin.RadioButton" parent="Widget.MaterialComponents.CompoundButton.RadioButton">
+        <item name="android:textColor">@color/vtmn_radio_button_text</item>
+        <item name="buttonTint">@color/vtmn_radio_button_item</item>
+        <item name="textInputStyle">@dimen/vtmn_line_height_body_2</item>
```

Comment:
Can't find this property on Material documentation (https://material.io/components/radio-buttons/android#theming-radio-buttons)
But we can specify the text appearance: https://material.io/components/radio-buttons/android#key-properties
Seems to be a better choice.

Reply:
Good catch, I was pretty sure it was working (and I was wrong) 👌